website/static/js/pages/faq-page.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Code for interaction in the support page
+ */
+
+'use strict';
+var $ = require('jquery');
+
+$(document).ready(function(){
+    /**
+     * handle expanding or collapsing individual support item with jQuery
+     * @param item {Object} jQuery object for the .support-item element
+     * @param turnOff {Boolean} true if we are closing and open item
+     * @private
+     */
+    function changeExpandState (item, turnOff) {
+        var body = item.children('.support-body');
+        var head = item.children('.support-head');
+        var icon = head.children('.fa');
+        if (turnOff){
+            body.slideUp();
```

Comment:
Minor: the default animation times for these slides are a bit slow. Might be worth speeding these up a bit.